Tabloid/Controllers/ReactionController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Tabloid.Models;
+using Tabloid.Repositories;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace Tabloid.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ReactionController : ControllerBase
+    {
+        private readonly IReactionRepository _reactionRepository;
+
+        public ReactionController(IReactionRepository reactionRepository)
+        {
+            _reactionRepository = reactionRepository;
+        }
+
+        [HttpGet]
+        public IActionResult Get()
+        {
+            return Ok();
+        }
+
+        //// GET api/<ReactionController>/5
+        //[HttpGet("{id}")]
+        //public string Get(int id)
+        //{
+        //    return "value";
+        //}
+
+        // POST api/<ReactionController>
+        [HttpPost]
+        public IActionResult Post(Reaction reaction)
+        {
+            _reactionRepository.Add(reaction);
+
+            return CreatedAtAction("Get", new { id = reaction.Id }, reaction);
+        }
+
+        // PUT api/<ReactionController>/5
+        //[HttpPut("{id}")]
+        //public void Put(int id, [FromBody] string value)
+        //{
+        //}
+
+        //// DELETE api/<ReactionController>/5
+        //[HttpDelete("{id}")]
+        //public void Delete(int id)
+        //{
+        //}
```

Comment:
Can you remove the commented lines? I think we're not supposed to have commented blocks of code in main

Reply:
Removed.

Reply:
looks good